### PR TITLE
[Onboarding] used dynamic styles for select option icon

### PR DESCRIPTION
**Pull request by xtina-starr**
This refactors the budget component to uses dynamic styling and fix the icon for the budget component

screen shot 2017-10-30 at 3 52 21 pm

### CODE REVIEW

**Comment by xtina-starr** [comment]
@yuki24 the linter is complaining about a file that wasn't modified. Have you seen this before? 🤔

**Comment by orta** [comment]
@xtina-starr me and @eessex just fixed it in #330

**Comment by orta** [comment]
It must have been introduced in #329 but couldn't figure it out, it's this commit that "fixes" it - https://github.com/artsy/reaction/pull/330/commits/3dbbf4c57a17e5f74d8748a1e2d76d79c4428c2d

**Comment by xtina-starr** [comment]
thanks for the info @orta. looks like #330 just got merged. should I rebase against master?

**Comment by orta** [comment]
yah

**Comment by xtina-starr** [comment]
💚

File: src/Components/Onboarding/SelectableToggle.tsx [hunk 1]
```diff
@@ -11,34 +11,39 @@ interface SelectableToggleProps {
...
+  isSelected: boolean
+}
+
+const iconContainer: StyledFunction<IconContainerProps> = styled.div
+
+const IconContainer = iconContainer`
+  width: 18px;
+  height: 18px;
+  background-color: black;
+  border-radius: 50%;
```
Comment (yuki24) [inline src/Components/Onboarding/SelectableToggle.tsx]: I would expect the inverse circle check mark to be part of the icon font set. Or, the `Icon` component should display it using the `follow-circle.is-following` icon and the CSS here, as if the font set has it. Alternatively, you could implement `CircleIcon` or some sort.

**Pull request merged**